### DISCUSSION TITLE
Space ROS Technical Meeting Agenda 2024-09-05

### DISCUSSION THREAD

**Discussion by Bckempa** [space-ros/space-ros#204] [General]
Previous Agenda: https://github.com/space-ros/space-ros/discussions/192
Active Milestone: [`jazzy-2024.10.0`](https://github.com/orgs/space-ros/projects/3/views/11)

### Agenda
- [x] Introduce any new attendees.
- [x] Summarize any PRs merged since last meeting.
  - https://github.com/space-ros/docker/pull/171#event-14136520957
  - Various [.github repo](https://github.com/space-ros/.github) project meta-config
- [ ] Review release follow-up issues and PRs
- [ ] Nominate work for next release
- [ ] Open next meeting discussion

**Comment by Bckempa** [comment]
We should consider adding a milestone or label for challenge submissions to make it easier to filter them when looking at PRs

**Comment by Bckempa** [comment]
Need someone who runs remote docker containers to test xauth: https://github.com/space-ros/docker/pull/175

**Comment by asimonov** [comment]
can discuss upgrade to Jazzy: https://github.com/space-ros/space-ros/issues/157

**Reply by Bckempa** [reply]
CI is running 👀 : https://github.com/space-ros/space-ros/pull/205

**Comment by Bckempa** [comment]
Next meeting agenda: https://github.com/space-ros/space-ros/discussions/213
